internal/state/actions.go

```diff
@@ -197,21 +192,14 @@ func (state *State) actionAddMessagesToMailbox(
 	return messageUIDs, nil
 }
 
-func (state *State) actionRemoveMessagesFromMailbox(
+// actionRemoveMessagesFromMailboxUnchecked is similar to actionRemoveMessagesFromMailbox, but it does not validate
+// the input for whether messages actually exist in the database or if the message set is empty.
+func (state *State) actionRemoveMessagesFromMailboxUnchecked(
```

Comment:
i'd like a comment here to describe in what cases this is actually useful -- describe why we don't need this validation in some cases

Reply:
Done.